savlatkhon-eshonov-1/task-2.ts

```diff
@@ -0,0 +1,16 @@
+class User {
+  _username:string;
+  _password:string;
+  constructor(username:string, password:string) {
+    this._username = username;
+    this._password = password;
+  }
+
+  validateLogin(inputPassword:string) {
```

Comment:
You must handle process of password check internally. Now you are exposing password check
` // Public method to validate login
public validateLogin(inputPassword: string): boolean {
// Abstracts the logic of password comparison
return this.checkPassword(inputPassword);
}

// Private method for password comparison
private checkPassword(inputPassword: string): boolean {
// Compare the input password with the actual password
return inputPassword === this.password;
}`